providers/aws/services/acm/acm_certificates_transparency_logs_enabled/acm_certificates_transparency_logs_enabled_test.py

```diff
@@ -0,0 +1,98 @@
+                acm_certificates_transparency_logs_enabled,
+            )
+
+            check = acm_certificates_transparency_logs_enabled()
+            result = check.execute()
+
+            assert len(result) == 1
+            assert result[0].status == "FAIL"
+
+    @mock_acm
+    def test_acm_certificate_without_logging(self):
+        # Generate ACM Client
+        acm_client = client("acm")
+        # Request ACM certificate
+        acm_client.request_certificate(
+            DomainName="test.com",
+            Options={"CertificateTransparencyLoggingPreference": "ENABLED"},
+        )
+        with mock.patch(
+            "providers.aws.services.acm.acm_certificates_transparency_logs_enabled.acm_certificates_transparency_logs_enabled.acm_client",
+            new=ACM(current_audit_info),
+        ):
+            # Test Check
+            from providers.aws.services.acm.acm_certificates_transparency_logs_enabled.acm_certificates_transparency_logs_enabled import (
+                acm_certificates_transparency_logs_enabled,
+            )
+
+            check = acm_certificates_transparency_logs_enabled()
+            result = check.execute()
+
+            assert len(result) == 1
+            assert result[0].status == "PASS"
+
+    @mock_acm
+    def test_acm_default_certificate(self):
+        # Generate ACM Client
+        acm_client = client("acm")
+        # Request ACM certificate
+        acm_client.request_certificate(
+            DomainName="test.com",
+        )
+        with mock.patch(
+            "providers.aws.services.acm.acm_certificates_transparency_logs_enabled.acm_certificates_transparency_logs_enabled.acm_client",
+            new=ACM(current_audit_info),
+        ):
+            # Test Check
+            from providers.aws.services.acm.acm_certificates_transparency_logs_enabled.acm_certificates_transparency_logs_enabled import (
+                acm_certificates_transparency_logs_enabled,
+            )
+
+            check = acm_certificates_transparency_logs_enabled()
+            result = check.execute()
+
+            assert len(result) == 1
+            assert result[0].status == "PASS"
+
+    @mock_acm
+    def test_bad_response(self):
+        mock_client = mock.MagicMock()
+        mock_client.credential_report = None
+        with mock.patch(
+            "providers.aws.services.acm.acm_certificates_transparency_logs_enabled.acm_certificates_transparency_logs_enabled.acm_client",
+            new=mock_client,
+        ):
+            # Test Check
+            from providers.aws.services.acm.acm_certificates_transparency_logs_enabled.acm_certificates_transparency_logs_enabled import (
+                acm_certificates_transparency_logs_enabled,
+            )
+
+            check = acm_certificates_transparency_logs_enabled()
+            result = check.execute()
+
+            assert len(result) == 0
```

Comment:
Is this tests working? You are creating a `credential_report` object for an ACM class that doesn't have this value in it.